jobspec/test-fixtures/dynamic-port-range.hcl

```diff
@@ -0,0 +1,20 @@
+job "foo" {
+  group "example" {
+    task "server" {
+      resources {
+        network {
+          mbits = 200
+          port "http" {}
+          port "https" {}
+          port "lb" {
+            static = "8889"
+          }
+          dynamic_port_range {
+              min = 4000
+              max = 5000
+          }
```

Comment:
I'm throwing ideas in the air, but could it be something like `static` but accepting a list of ranges.

```hcl
port "http" {}
port "https" {
  ranges = ["443","8443","4000-5000"]
}
port "lb" {
  static = "8889"
}
```